src/Mod/Draft/DraftSnap.py

```diff
@@ -281,7 +315,7 @@ def snapToObject(self, lastpoint, active, constrain, eline, point, oldActive):
             parent = obj
             subname = self.snapInfo['Component']
         if not obj:
-            self.spoint = cstr(point)
+            self.spoint = cstr(point) # BUG: cstr is not defined ??
```

Comment:
cstr is defined above, in line 175 above, but it doesn't seem to be used at the moment because it causes no error.